### ISSUE TITLE
SET.KEYS values always set to 1

### ISSUE THREAD

**Issue by GrayTShirt** [bolo/bolo#80]
This code may be the issue:
https://github.com/bolo/bolo/blob/847a7bd1ed313e25673010d5f49524490d623f65/src/bolo_pdu.c#L122-L130

**Comment by jhunt** [comment]
The collectors aren't sending the right format.

From my test lab, I see this:

```
$ linux | grep KEY
KEY localhost:fs:/var/vcap/data /dev/mapper/ubuntu--vg-root
KEY localhost:dev:/dev/mapper/ubuntu--vg-root /var/vcap/data
KEY localhost:fs:/warden-cpi-dev /dev/mapper/ubuntu--vg-root
KEY localhost:dev:/dev/mapper/ubuntu--vg-root /warden-cpi-dev
```

when I should be seeing this:

```
% linux | grep KEY
KEY localhost:fs:/var/vcap/data=/dev/mapper/ubuntu--vg-root
KEY localhost:dev:/dev/mapper/ubuntu--vg-root=/var/vcap/data
KEY localhost:fs:/warden-cpi-dev=/dev/mapper/ubuntu--vg-root
KEY localhost:dev:/dev/mapper/ubuntu--vg-root=/warden-cpi-dev
```

(note the missing `=` between key and value in the first output block)

**Comment by jhunt** [comment]
This is fixed in bolo/bolo-collectors@c972849
